paper-slider.html

```diff
@@ -350,14 +350,13 @@
       this._minx = - this._startx;
       this._maxx = this._w - this._startx;
       this.$.sliderKnob.classList.add('dragging');
+
       this._setDragging(true);
-      e.preventDefault();
```

Comment:
Maybe the reason this doesn't work is that `e` is a synthetic `CustomEvent`, and not the actual MouseEvent / TouchEvent that caused the gesture?